.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,37 @@
-*To help us understand & test your amazing contribution, please include the below information with your PR. 🙏 Thanks!*
+<!-- Employees: Please use Clubhouse's "Open PR" button from the relevant story or include links to relevant Clubhouse stories in your branch name, commit messages, or pull request comments. Do not add links to your pull request description, they will be ignored. https://help.clubhouse.io/hc/en-us/articles/207540323-Using-The-Clubhouse-GitHub-Integration -->
 
-
-### Closes #ISSUE_NUMBER_GOES_HERE
+<!-- Employees: Delete this section. -->
+## Closes #ISSUE_NUMBER_GOES_HERE
 
 🎉 🎉 🎉
 
-### Description
+## Description
 
 ......
 
-
-### Changes
+## Changes
 
 * High level
 * changes that
 * you made
 
+### Reusable Code/Components
 
-### Steps to Test
+<!-- Preferably, include automated tests instead. -->
+## Steps to Test
 
 1. Steps
 2. to
 3. test
 
+<!-- Contributors: Delete this section. -->
+## [Zeplin Design](https://app.zeplin.io/project/5b0334f5e91e8c481645ad56)
+<!-- Upload screenshots here. -->
 
-### Screenshots
+## Quality Assurance
 
-...only if applicable...
+- [ ] The branch name is in lowercase-kebab-case with no prefix (unless it was created from Clubhouse)
+- [ ] The base branch is develop or gau (no nested branches)
+- [ ] This is related to a maximum of one Clubhouse story or GitHub issue
+- [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
```

Comment:
This should be handled by our linter. I don't see why we need a reminder here.

Reply:
Our linter allows unsafe types like `any` and `{}` if they're used explicitly. This is still dangerous, but changing the linter setting would require us to fix a lot of existing TypeScript types. In this case it's easier for us to catch the issue in manual reviews until we can fix the overall type safety in the future.

Reply:
Ok.
An alternative in the future would be to run linting only on changed files, which would allow us to gradually enforce better standards

Reply:
Our linter already only runs on changed file. TypeScript does not support this.

Reply:
I just checked and our current TSLint setup only runs on changed files. Unfortunately this isn't possible with TypeScript because the compiler needs to run with the entire project to determine types.

Reply:
Didn't think of that though I believe that tsc can resolve the imports and find the used types.

Reply:
The problem is that as far as I'm aware, TypeScript needs to typecheck all imported files to understand their types, and typechecking can't be enabled or disabled per file because of the way type errors are immplemented with the typechecker. The TypeScript team seems to be against implementing this, and suggests that users should migrate their code in one fell swoop.

So ideally, we should improve as much of these unsafe type practices as possible (any, disable, overly wide types, casts, etc.) and then make our linter rules more strict (when we have time).